src/main/java/Main.java

```diff
@@ -43,33 +47,32 @@ public static void readFile() {
                
                if(applianceName.contains("AC")) {
                    // add event to eventQueue of AirConditioner
+                   //System.out.println("READ: AC");
+                   //System.out.println(event.getStatus());
+                   //System.out.println(event.getTime());
+                   //System.out.println("=========");
                    airConditioner.addEvent(event);
-               } else if(applianceName.contains("Oven")) {
+               } else if(applianceName.contains("OVEN")) {
```

Comment:
Adding a new appliance makes it mandatory to add it to this if-else section. Try to follow [open-closed principle](https://en.wikipedia.org/wiki/Open/closed_principle)